pkg/apis/workflow/v1alpha1/workflow_types.go

```diff
@@ -568,7 +568,7 @@ type Template struct {
 	// Optional duration in seconds relative to the StartTime that the pod may be active on a node
 	// before the system actively tries to terminate the pod; value must be positive integer
 	// This field is only applicable to container and script templates.
-	ActiveDeadlineSeconds *intstr.IntOrString `json:"activeDeadlineSeconds,omitempty" protobuf:"bytes,21,opt,name=activeDeadlineSeconds"`
+	ActiveDeadlineSeconds *intstr.IntOrString `json:"activeDeadlineSeconds,string,omitempty" protobuf:"bytes,21,opt,name=activeDeadlineSeconds"`
```

Comment:
I did non know about this! Neat! 

@sarabala1979 @whynowy @jessesuen @alexmt 

https://pkg.go.dev/encoding/json

Reply:
I didn't see any change in the codegenerator.

Reply:
Maybe need to run “make codegen”